automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/user/SignInPage.scala

```diff
@@ -55,7 +56,7 @@ class SignInPage(val baseUrl: String)(implicit webDriver: WebDriver) extends Fir
     val initialWindowHandles = windowHandles
 
     signInButton.doClick()
-    await condition (windowHandles.size > 1)
+    await condition (windowHandles.size == 2)
```

Comment:
open windows size must be exact 2. if more, sign of something went wrong.

Reply:
Curious why there would be more than 2. Is this something you've seen?

Reply:
No. a precautionary step.